ush/cam/evs_href_prepare.sh

```diff
@@ -339,9 +339,9 @@ if [ "$data" = "gfs_prepbufr" ] ; then
       done
    else
       if [ "$SENDMAIL" = "YES" ] ; then
-	 echo "Warning:  No GFS Prepbufr data available for ${VDATE}"     
+	 echo "WARNING:  No GFS Prepbufr data available for ${VDATE}"     
          export subject="GFS Prepbufr Data Missing for EVS ${COMPONENT}"
-         echo "Warning:  No GFS Prepbufr data available for ${VDATE}" > mailmsg
+         echo "WARNING:  No GFS Prepbufr data available for ${VDATE}" > mailmsg
          echo Missing file is $COMINobsproc/gdas.${vday}/18/atmos/gdas.t18z.prepbufr  >> mailmsg
```

Comment:
We need to move this WARNING statement outside the SENDMAIL block, including the full path.